src/integration/common/analytics.canonical.js

```diff
@@ -0,0 +1,16 @@
+const { canonical } = global;
+
+export default () => {
+  it('I can see the data collected with ATI analytics', () => {
+    process.env.SIMORGH_ATI_BASE_URL = 'https://logws1363.ati-host.net?';
```

Comment:
Do we need to ensure that this sort of thing is reset between tests/suites to avoid interactions?

Reply:
🤔 I'll investigate. Jest does a tear down after each test suite is run but I don't exactly know what it tears down

Reply:
Cheers! I did notice that elsewhere we call `resetModules` (e.g. https://github.com/bbc/simorgh/blob/0fbaf09e9870111b177d89344ea592624951f011/src/app/containers/ATIAnalytics/amp/index.test.jsx#L12) but I don't know if that is actually necessary (or sufficient).

Reply:
Do we still need to clear this variable after the test is completed?